code/modules/power/solar.dm

```diff
@@ -203,6 +204,7 @@
 	w_class = ITEM_SIZE_BULKY // Pretty big!
 	anchored = FALSE
 	price_tag = 100
+	matter = list(MATERIAL_STEEL = 5, MATERIAL_PLASTIC = 5, MATERIAL_PLASTEEL = 2, MATERIAL_SILVER = 1)
```

Comment:
```suggestion
	matter = list(MATERIAL_STEEL = 5, MATERIAL_PLASTIC = 5, MATERIAL_SILVER = 1)
```
Plasteel is probably a bit too much.
There should be a way to get solar panels outside of autolathe - like guild. Or technomancers could spawn with one of those "Going green!" crates.

Reply:
> Solar panels are made of tempered glass, which is sometimes called toughened glass.

I would suggest replacing plasteel with 5 glass instead, maybe reducing the steel/plastic price too

Reply:
dont forgot, this just 1 panel, you need a lot for generate.

Reply:
Sheets are inserted in a separate step...

Reply:
> Sheets are inserted in a separate step...

he mean, replace plasteel in recipe on glass. Btw accept PR please )